ash/src/extensions/khr/video_queue.rs

```diff
@@ -0,0 +1,244 @@
+        &self,
+        command_buffer: vk::CommandBuffer,
+        end_coding_info: &vk::VideoEndCodingInfoKHR<'_>,
+    ) {
+        (self.fp.cmd_end_video_coding_khr)(command_buffer, end_coding_info)
+    }
+
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCreateVideoSessionKHR.html>
+    #[inline]
+    #[doc(alias = "vkCreateVideoSessionKHR")]
+    pub unsafe fn create_video_session(
+        &self,
+        create_info: &vk::VideoSessionCreateInfoKHR<'_>,
+        allocation_callbacks: Option<&vk::AllocationCallbacks<'_>>,
+    ) -> VkResult<vk::VideoSessionKHR> {
+        let mut video_session = mem::MaybeUninit::uninit();
+        (self.fp.create_video_session_khr)(
+            self.handle,
+            create_info,
+            allocation_callbacks.as_raw_ptr(),
+            video_session.as_mut_ptr(),
+        )
+        .assume_init_on_success(video_session)
+    }
+
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCreateVideoSessionParametersKHR.html>
+    #[inline]
+    #[doc(alias = "vkCreateVideoSessionParametersKHR")]
+    pub unsafe fn create_video_session_parameters(
+        &self,
+        create_info: &vk::VideoSessionParametersCreateInfoKHR<'_>,
+        allocation_callbacks: Option<&vk::AllocationCallbacks<'_>>,
+    ) -> VkResult<vk::VideoSessionParametersKHR> {
+        let mut video_session_parameters = mem::MaybeUninit::uninit();
+        (self.fp.create_video_session_parameters_khr)(
+            self.handle,
+            create_info,
+            allocation_callbacks.as_raw_ptr(),
+            video_session_parameters.as_mut_ptr(),
+        )
+        .assume_init_on_success(video_session_parameters)
+    }
+
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkDestroyVideoSessionKHR.html>
+    #[inline]
+    #[doc(alias = "vkDestroyVideoSessionKHR")]
+    pub unsafe fn destroy_video_session(
+        &self,
+        video_session: vk::VideoSessionKHR,
+        allocation_callbacks: Option<&vk::AllocationCallbacks<'_>>,
+    ) {
+        (self.fp.destroy_video_session_khr)(
+            self.handle,
+            video_session,
+            allocation_callbacks.as_raw_ptr(),
+        )
+    }
+
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkDestroyVideoSessionParametersKHR.html>
+    #[inline]
+    #[doc(alias = "vkDestroyVideoSessionParametersKHR")]
+    pub unsafe fn destroy_video_session_parameters(
+        &self,
+        video_session_parameters: vk::VideoSessionParametersKHR,
+        allocation_callbacks: Option<&vk::AllocationCallbacks<'_>>,
+    ) {
+        (self.fp.destroy_video_session_parameters_khr)(
+            self.handle,
+            video_session_parameters,
+            allocation_callbacks.as_raw_ptr(),
+        )
+    }
+
+    /// Retrieve the number of elements to pass to [`get_video_session_memory_requirements()`][Self::get_video_session_memory_requirements],
```

Comment:
```suggestion
    /// Retrieve the number of elements to pass to [`get_video_session_memory_requirements()`][Self::get_video_session_memory_requirements()].
```

Though let's double-check other functions - I think none terminate this sentence.